expression/testdata/partition_pruner_in.json

```diff
@@ -2,23 +2,22 @@
   {
     "name": "TestHashPartitionPruner",
     "cases": [
-      // Point Select.
       "explain select * from t1 where id = 7 and a = 6",
       "explain select * from t3 where id = 9 and a = 1",
       "explain select * from t2 where id = 9 and a = -110",
       "explain select * from t1 where id = -17",
       "explain select * from t2 where id = a and a = b and b = 2",
-      // Join.
       "explain select * from t1 join t2 on (t1.id = t2.id) where t1.id = 5 and t2.a = 7",
       "explain select * from t1 left join t2 on t1.id = 1 and t2.a = 2 where t2.id = 7",
       "explain select * from t2 join t1 on t1.id = t2.id and t2.a = t1.id and t2.id = 12",
-      // Negtive cases.
       "explain select * from t1 left join t2 on true where t1.a = 1 and false",
       "explain select * from t1 left join t2 on true where t1.a = 1 and null",
       "explain select * from t1 left join t2 on true where t1.a = null",
-      // Case with date.
       "explain select * from t4 where d = '2019-10-07 10:40:00' and a = 1",
-      "explain select * from t5 where d = '2019-10-07'"
+      "explain select * from t5 where d = '2019-10-07'",
+      "explain select * from t6 where a is null",
```

Comment:
add a case for `explain select * from t6 where b is null`

Reply:
done, PTAL